docs/source/release_notes.rst

```diff
@@ -20,6 +20,7 @@ Release Notes
         * Use new version of pull request Github Action (:pr:`1234`)
         * Upgrade pip during featuretools[complete] test (:pr:`1236`)
         * Migrated CI tests to github actions (:pr:`1226`)
+        * Remove CI tests from CircleCI (:pr:`1237`)
```

Comment:
instead of adding a second release note, let's include this PR number with the "Migrated CI tests to github actions"